README.md

```diff
@@ -1,21 +1,63 @@
-# 🇨🇦 🇬🇹 🇰🇷 🇱🇧 Olympick app 🏴󠁧󠁢󠁷󠁬󠁳󠁿 🇵🇰 🇮🇷 🇬🇧 🇨🇴 
+# 🇨🇦 🇬🇹 🇰🇷 🇱🇧 🇯🇵 Olympick 🏴󠁧󠁢󠁷󠁬󠁳󠁿 🇵🇰 🇮🇷 🇬🇧 🇨🇴 
 
-Currently there is no way to create a viewing schedule for the Olympics that only includes specific sports.
+Currently there is no way to create a viewing schedule for the Olympics that only include specific sports. Olympick aims to solve this problem by helping people create their own personalised schedule for watching events of their favourite sports in UK time. Events already watched or no longer of interest can then be removed from schedules and/or new events can be added.
 
-This app allow a user to : <br/>
- <li>input their favourite Olympic sport's name and get all the schedules of it.</li><br/>
- <li>input their favourite Olympic sport's name  and get the schedules of specific event of it.</li><br/>
+Olympick runs in the terminal and in the browser. Please see installation instructions below.
 
-# Tools
+<br>
 
-<li> 2020 Olympic Public API : (https://olypi.com/) </li>
-<li> SQL databases</li>
-<li> Python language</li>
-<li> Python Libraries : <strong>Flask</strong>,<strong>Collections</strong>,etc </li>
-<li>Tests : Unit tests and mock tests</li>
```

Comment:
@melanieet Could you please mention the user can go through a journey in the terminal or in the browser? 
The user can go through the journey in the browser by running the app.py file and all of these explanations are for the browser (you added them here)
or they just can go through the journey in the terminal by running the main.py file